tests.nim

```diff
@@ -337,6 +338,15 @@ suite "pattern walking / listing":
         "temp" / "shallow.nim"
       ])
 
+    test "`IgnoreCase` enables case insensitive matching":
+      let o = defaultGlobOptions + {IgnoreCase}
+      check seqsEqual(toSeq(walkGlob("TEMP/**", options = o)), @[
+        "temp" / "deep" / "dir" / "file.nim",
```

Comment:
nit: maybe more readable via:
`"temp/deep/dir/file.nim".unixToNativePath` ?

Reply:
Hmm might even be nice to put a template in here to do `p"temp/deep/dir/file.nim"`